_posts/2016-05-16-rust-at-one-year.md

```diff
@@ -0,0 +1,268 @@
+[Ruby]: http://diesel.rs/
+
+And yesterday, Rust turned one year old.
+
+### Rust in numbers
+
+A lot has happened in the last 365 days:
+
+- 11,894 [commits] by 702 contributors added to the core repository;
+- 88 [RFCs] merged;
+- 18 compiler targets introduced;
+- 9 releases shipped;
+- 1 year of [stability delivered].
+
+On an **average week** this year, the Rust community merged two RFCs and
+published 53 brand new [crates]. Not a single day went by without at least one
+new Rust library hitting the central package manager. And Rust topped the
+"[most loved] language" in this year's StackOverflow survey.
+
+> Speaking of numbers: we recently launched a [survey] of our own, and want to
+> hear from you whether you are an old hat at Rust, or have never used it.
+
+One place where our numbers are not where we want them to be: community
+diversity. We've had ongoing local outreach efforts, but the Rust community team
+will soon be launching a coordinated, global effort following the [Bridge] model
+(e.g. RailsBridge). If you want to get involved, or have other ideas for
+outreach, please let [the community team] know.
+
+[survey]: http://blog.rust-lang.org/2016/05/09/survey.html
+
+[most loved]: https://stackoverflow.com/research/developer-survey-2016
+
+[commits]: https://github.com/rust-lang/rust/commits/master
+[RFCs]: https://github.com/rust-lang/rfcs
+[stability delivered]: http://blog.rust-lang.org/2014/10/30/Stability.html
+[crates]: https://crates.io/
+
+[Bridge]: http://bridgefoundry.org/
+[the community team]: mailto:community-team@rust-lang.org
+
+### Rust in production
+
+This year saw more companies [betting on Rust]. Each one has a story, but two
+particularly resonated.
+
+[betting on Rust]: https://www.rust-lang.org/friends.html
+
+First, there's Dropbox. For the last several years, the company has been
+secretively working on a move
+[away from AWS and onto its own infrastructure][dropbox]. The move, which is now
+complete, included developing custom-build hardware and the software to drive
+it. While much of Dropbox's back-end infrastructure is historically written in
+Go, for some key components the memory footprint and lack of control stood in
+the way of achieving the server utilization they were striving for. They rewrote
+those components in Rust.  In the [words of Jamie Turner][dropbox quote], a lead
+engineer for the project, "the advantages of Rust are many: really powerful
+abstractions, no null, no segfaults, no leaks, yet C-like performance and
+control over memory."
+
+[dropbox]: http://www.wired.com/2016/03/epic-story-dropboxs-exodus-amazon-cloud-empire/
+[dropbox quote]: https://news.ycombinator.com/item?id=11283688
+
+Second, there's Mozilla. They've long been developing [Servo] as a research
+browser engine in Rust, but their first *production* Rust code shipped through a
+different vehicle: Firefox. In Firefox 45, without any fanfare, Rust code for
```

Comment:
nit: Firefox needs more emph.  It's the punchline of the sentence.  Could use "firefox itself" or bold.